frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/embedding/common.embedding.unit.spec.tsx

```diff
@@ -4,56 +4,33 @@ import { act, screen, within } from "__support__/ui";
 import {
   createMockSettingDefinition,
   createMockSettings,
+  createMockTokenFeatures,
```

Comment:
Please see this comment https://github.com/metabase/metabase/pull/47770#discussion_r1755300862